jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -1371,13 +1372,8 @@ ecma_string_get_chars (const ecma_string_t *string_p, /**< ecma-string */
       JERRY_ASSERT (ECMA_STRING_GET_CONTAINER (string_p) == ECMA_STRING_CONTAINER_MAGIC_STRING_EX);
 
       size = lit_get_magic_string_ex_size (string_p->u.magic_string_ex_id);
-      length = 0;
-
-      if (is_ascii_p != NULL)
-      {
-        length = lit_utf8_string_length (lit_get_magic_string_ex_utf8 (string_p->u.magic_string_ex_id),
-                                         lit_get_magic_string_ex_size (string_p->u.magic_string_ex_id));
-      }
+      length = lit_utf8_string_length (lit_get_magic_string_ex_utf8 (string_p->u.magic_string_ex_id),
+                                       lit_get_magic_string_ex_size (string_p->u.magic_string_ex_id));
```

Comment:
As I see `size` can be used instead of `lit_get_magic_string_ex_size (string_p->u.magic_string_ex_id)` like:

`length = lit_utf8_string_length (lit_get_magic_string_ex_utf8 (string_p->u.magic_string_ex_id),
                                                  size);` 

There would be one less function call.

Reply:
good catch